source/github-helpers/get-pr-info.ts

```diff
@@ -30,6 +32,9 @@ export default async function getPrInfo(base: string, number = getConversationNu
 						aheadBy
 					}
 				}
+				headRepository {
+					viewerPermission
+				}
```

Comment:
If you have time it would be nice to extract this whole query to its own .gql file, it just needs to accept a couple of variables. See other .gql files